### DISCUSSION TITLE
loxilb non-privileged

### DISCUSSION THREAD

**Discussion by slzz123** [loxilb-io/loxilb#654] [Q&A]
Dear Loxilb Developers and Community,

Could you kindly provide insights or guidance on whether it's possible to run Loxilb in a non-privileged user environment? We are interested in understanding any specific configurations or considerations that might be necessary to achieve this.

Thank you very much for your time and assistance.

BR

**Comment by TrekkieCoder** [comment]
You can check non-privileged mode manifest [here](https://github.com/loxilb-io/kube-loxilb/blob/main/manifest/service-proxy/loxilb-service-proxy-non-privileged.yml) . Also some changes required #678
